<launch>
    <arg name="param_file" default="/home/pepper/citk/systems/pepper-robocup-nightly/share/pepper_perception_configs/vision/clf_perception/clf_perception_face_example.yaml" />
    <rosparam command="load" file="$(arg param_file)"/>
    
    <arg name="imageToPoseClient_topic" default="/clf_perception_vision/get_pose_from_image" />
    <arg name="bounding_box_topic" default="/people_tracker/people/extended" />
    <arg name="publish_topic" default="/clf_perception_vision/people/head_extended" />
    <arg name="face_frontal" default="/home/pepper/citk/systems/pepper-robocup-nightly//share/clf_perception_vision/data/haar/haarcascade_frontalface_default.xml" />
    <arg name="face_profile" default="/home/pepper/citk/systems/pepper-robocup-nightly//share/clf_perception_vision/data/haar/haarcascade_profileface.xml" />
    <arg name="machine" default="localhost" />
    <arg name="user" default="" />

    <machine name="$(arg machine)" address="$(arg machine)" env-loader="$(optenv ROS_ENV_LOADER )" user="$(arg user)" default="true"/>
    
    <node pkg="clf_perception_vision" type="clf_perception_face" name="clf_perception_face" output="screen" respawn="true">
        <param name="imageToPoseClient_topic" value="$(arg imageToPoseClient_topic)"/>
        <param name="bounding_box_topic" value="$(arg bounding_box_topic)" type="string"/>
        <param name="publish_topic" value="$(arg publish_topic)" type="string"/>
        <param name="face_frontal" value="$(arg face_frontal)" type="string"/>
        <param name="face_profile" value="$(arg face_profile)" type="string"/>
    </node>
</launch> 